docs/baremetal-operator-ironic-deployment.md

```diff
@@ -0,0 +1,104 @@
+# Running Bare Metal Operator with/without Ironic
```

Comment:
Maybe this file should just be called `deploying.md`?

Reply:
Hi! Thanks for a review, updated the name of the document to `deploying`.